Shear/HistoryItem.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  HistoryItem.swift
+//  Shear
+//
+//  Created by Andreas Kompanez on 23.02.20.
+//  Copyright © 2020 Andreas Kompanez. All rights reserved.
+//
+
+import Foundation
+
+protocol StringRepresentable {
+    var string: String { get }
+}
+
+enum HistoryItemKind: Hashable, Equatable {
+    case plainText(String)
+}
+
+struct HistoryItem: Hashable {
+    let kind: HistoryItemKind
+    
+    init(kind: HistoryItemKind) {
+        self.kind = kind
+    }
+}
+
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/kmpnz/Shear/blob/895e50bbb85826ea77037e4a1c1e5468d960c4fc/Shear/HistoryItem.swift#L27"></span>Files should have a single trailing newline.<br />
<code>trailing_newline</code> <code>HistoryItem.swift:27</code></del></td>
    </tr>
  </tbody>
</table>